test/metabase/events/revision_test.clj

```diff
@@ -38,7 +38,8 @@
    :cache_ttl              nil
    :query_type             :query
    :table_id               (mt/id :categories)
-   :visualization_settings {}})
+   :visualization_settings {}
+   :dataset_query_metrics_v2_migration_backup nil})
```

Comment:
Same, why not just change the assertions to `=?` so a) we don't mess up the indentation, and b) next time we add a new column we don't have to update this test again